### ISSUE TITLE
clickListener for chords

### ISSUE THREAD

**Issue by frauber84** [paulrosen/abcjs#715]
Could you add a clickListener selectType for chords (those set with "Em") ?

**Comment by paulrosen** [comment]
Yes, that looks like an oversight. I'll add it to my bug list but I don't know when I'll get to it.

**Comment by frauber84** [comment]
Thanks! I might try to write a pull request on that, could you share me some directions where should I look at the code to add that feature?

**Comment by paulrosen** [comment]
Probably https://github.com/paulrosen/abcjs/blob/main/src/write/selection.js
